core/editable.js

```diff
@@ -59,8 +59,26 @@
 		},
 
 		proto: {
-			focus: function() {
-
+			/**
+			 * Moves the selection focus to this editable element.
+			 *
+			 * Takes optional param `focusOptions` which holds property `preventScroll`. If set to true will prevent scrolling on focus.
+			 * Another property of `focusOptions` is `defer`. If set to true focus will be delayed by 100ms.
+			 *
+			 * Example:
+			 * ```javascript
+			 * editable.focus( { preventScroll: true } ); // Focuses but prevents scrolling.
+			 * editable.focus( { preventScroll: false } ); // Focuses without preventing scroll.
+			 * editable.focus( { defer: false } ); // Focuses after 100ms.
+			 * editable.focus(); // Focuses without preventing scroll.
+			 * ```
+			 *
+			 * @param {Object} [focusOptions] Focus options since **4.11.0**.
+			 * @param {Boolean} [focusOptions.preventScroll] Whenever after focus scroll should be prevented.
+			 * @param {Boolean} [focusOptions.defer] Whether to asynchronously defer the
```

Comment:
Actually I don't think we need it, but as this method calls element.focus with given focus options we would have to explicitly `delete focusOptions.defer`. Otherwise it would work as an undocumented feature. I'm for keeping it. OTOH as I'm moving `focusOptions` docs to dedicated type to keep it consistent - each focus method should allow same `focusOptios` type holding same properties.